src/js/views/globals/modal/loading/loading_modal.scss

```diff
@@ -0,0 +1,98 @@
+@keyframes blow-up-modal {
+  0% {
+    opacity: 0;
+    transform: scale(0.5) translateY(30%);
+  }
+
+  100% {
+    opacity: 1;
+    transform: scale(1) translateY(0%);
+  }
+}
+
+@keyframes progress-loader {
+  5% {
+    transform: translate3d(0, 0, 0);
+  }
+
+  10% {
+    width: 100px;
+  }
+
+  15% {
+    transform: translate3d(192px, 0, 0);
+    width: 64px;
+  }
+
+  20% {
+    width: 100px;
+  }
+
+  25% {
+    transform: translate3d(0, 0, 0);
+    width: 64px;
+  }
+
+  30% {
+    width: 100px;
+  }
+
+  35% {
+    transform: translate3d(200px, 0, 0);
+    width: 64px;
+  }
+
+  40% {
+    width: 100px;
+  }
+
+  45% {
+    transform: translate3d(0, 0, 0);
+    width: 64px;
+  }
+
+  100% {
+    width: 256px;
+  }
+}
+
+.loading-modal__wrapper {
+  position: absolute;
+}
+
+.loading-modal__content {
+  background-color: $white;
+  border-radius: 4px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  min-height: 262px;
+  text-align: center;
+  width: 524px;
+
+  &.is-animated {
+    animation: blow-up-modal 0.5s cubic-bezier(0.165, 0.84, 0.44, 1) forwards;
+    opacity: 0;
+  }
+}
+
+.loading-modal__text {
```

Comment:
I think we can probably dump this class.  Add it to `__content` if needed?

Reply:
Removed `.loading-modal__text` and moved `line-height: 1` to `.loading-modal__content`. Without `line-height: 1`, there's some unnecessary spacing between the text and progress bar.